card-game/src/components/Modal.jsx

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
+
+
+export default function Modal({success, setSuccess}) {
+  return (
+    <ModalBox success={success}>
+        <Content>
+            <ModalText>🎊축하합니당 숭숭박사시네용🎊</ModalText>
+            <ModalButton onClick={() => setSuccess(false)}>게임으로 돌아가기 </ModalButton>
+        </Content>
+    </ModalBox>
+  )
+}
+
+const ModalBox = styled.section`
+    position: absolute;
+    width:100vw;
+    height:100vh;
+    z-index: 999;
+
+    justify-content: center;
+    align-items: center;
+
+    background-color: ${({ theme }) => theme.colors.backgroundBlack};
+
+    display : ${(props) => (props.success ? "flex" : "none")};
```

Comment:
요고 그냥 이렇게 써두 된답니당!
```suggestion
    display : ${(success) => (success ? "flex" : "none")};
```
그리고 보통 상태를 나타내는 경우에는 변수명을 isSuccess 이렇게 쓰는 편이라고 해요! 이건 자유지만 참고!!